openvdb/openvdb/math/Maps.h

```diff
@@ -293,14 +293,9 @@ class OPENVDB_API MapRegistry
 ////////////////////////////////////////
 
 
-/// @note Macro to use a final specifier from ABI=8 onwards.
-#define OPENVDB_MAP_CLASS_SPECIFIER final
-#define OPENVDB_MAP_FUNC_SPECIFIER final
-
 /// @brief A general linear transform using homogeneous coordinates to perform
 /// rotation, scaling, shear and translation
-/// @note This class is marked final with ABI=8
-class OPENVDB_API AffineMap OPENVDB_MAP_CLASS_SPECIFIER: public MapBase
+class OPENVDB_API AffineMap final: public MapBase
```

Comment:
why is it important to prevent inheriting from AffineMap?

Reply:
Performance. By preventing inheritance, you can ensure that when you have an AffineMap object, calling one of the methods on it can be de-virtualized which eliminates the virtual function cost and allows for inlining. 

This feature has been enabled for the last couple of years incidentally - this is just tidying up and removing the macro.